csv.yml

```diff
@@ -1,2 +0,0 @@
-:csv:
-  :enable_module: true
```

Comment:
isn't this file needed?

i chatted with tom and the other files in ./config/ really aren't needed at all.